src/celeritas/optical/OpticalPropertyData.hh

```diff
@@ -15,6 +15,21 @@
 
 namespace celeritas
 {
+//---------------------------------------------------------------------------//
+/*!
+ * Material dependent scintillation spectra.
+ *
+ * Support upto 3-components: [fast, medium, slow]
+ */
+struct ScintillationSpectra
```

Comment:
An instance of this class is a spectrum (singular) right?
```suggestion
struct ScintillationSpectrum
```

Reply:
Well, the actual emission spectrum is a convolution of one or more individual spectrum (so, I use the plural), but I agree that spectrum is more generic.  Also changed spectra to spectrum accordingly.